filebeat/docs/fields.asciidoc

```diff
@@ -30,11 +30,13 @@ grouped in the following categories:
 * <<exported-fields-logstash>>
 * <<exported-fields-mongodb>>
 * <<exported-fields-mysql>>
+* <<exported-fields-netflow>>
```

Comment:
The x-pack fields aren't in the docs in 6.x. This will need a "backport" (using quotes b/c it may be easier to just copy over the parts we need since they are spread across commits in master).